lib/lrmd/lrmd_client.c

```diff
@@ -2029,6 +2029,32 @@ lrmd_api_exec(lrmd_t * lrmd, const char *rsc_id, const char *action, const char
 }
 
 static int
+lrmd_api_exec_alert(lrmd_t * lrmd, const char *alert_id,
+              int timeout,      /* ms */
+              enum lrmd_call_options options, lrmd_key_value_t * params)
+{
+    int rc = pcmk_ok;
+    xmlNode *data = create_xml_node(NULL, F_LRMD_RSC);
+    xmlNode *args = create_xml_node(data, XML_TAG_ATTRS);
+    lrmd_key_value_t *tmp = NULL;
+
+    crm_xml_add(data, F_LRMD_ORIGIN, __FUNCTION__);
+    crm_xml_add(data, F_LRMD_RSC_ID, alert_id);
+    crm_xml_add(data, F_LRMD_RSC_ACTION, "start");
+    crm_xml_add_int(data, F_LRMD_TIMEOUT, timeout);
```

Comment:
I confirmed it, but F_LRMD_RSC_ACTION was necessary.
I make modifications to set it in create_lrmd_cmd().